@jbryan I get this parse error

``` pythonstack
 ======================================================================
ERROR: testParse (agamemnon.tests.test_rdf.GraphMemoryTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/trhowe/src/globusonline/agamemnon/agamemnon/tests/test_rdf.py", line 460, in testParse
    self.graph1.serialize("serialized.rdf")
  File "/Users/trhowe/src/globusonline/agamemnon/vendor/lib/python2.7/site-packages/rdflib/graph.py", line 781, in serialize
    serializer.serialize(stream, base=base, encoding=encoding, **args)
  File "/Users/trhowe/src/globusonline/agamemnon/vendor/lib/python2.7/site-packages/rdflib/plugins/serializers/rdfxml.py", line 49, in serialize
    write('<?xml version="1.0" encoding="%s"?>\n' % self.encoding)
  File "/Users/trhowe/src/globusonline/agamemnon/vendor/lib/python2.7/site-packages/rdflib/plugins/serializers/rdfxml.py", line 46, in <lambda>
    self.write = write = lambda uni: stream.write(uni.encode(encoding, 'replace'))
IOError: File not open for writing
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,11 +1,10 @@
-agamemnon==0.3.3.0
-isodate==0.4.7
-nose==1.1.2
+PyYAML==3.10
+isodate==0.4.8
 ordereddict==1.1
 pycassa==1.4.0
+pyes==0.16.0
 pyparsing==1.5.6
 python-dateutil==1.5
-rdfextras==0.1
-rdflib==3.2.0
+rdfextras==0.2
+rdflib==3.2.1
 thrift==0.8.0
-wsgiref==0.1.2

--- a/setup.py
+++ b/setup.py
@@ -6,7 +6,15 @@ here = os.path.abspath(os.path.dirname(__file__))
 README = open('README.md').read()
 CHANGES = open('CHANGES.txt').read()
 
-requires = ['pycassa==1.4.0', 'thrift', 'pyes', 'setuptools','ordereddict','rdflib','rdfextras','python-dateutil==1.5', 'pyyaml']
+requires = ['pycassa==1.4.0',
+    'thrift==0.8.0',
+    'pyes==0.16.0',
+    'ordereddict==1.1',
+    'rdflib==3.2.1',
+    'rdfextras==0.2',
+    'python-dateutil==1.5',
+    'pyyaml==3.10']
+
 tests_requires = requires + ['nose', 'mock']
 
 setup(name='agamemnon',

